.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: python
+python: 3.5
```

Comment:
Ahah. I did not realize that. In that case, can you add a `basepython` line to the pep8 test so it runs using Python 2 instead of the default Python 3.5?